meetings/2021-03-15.md

```diff
@@ -0,0 +1,150 @@
+# 2021-03-15 Authentication Panel
+
+## Meeting
+ * Call: https://meet.jit.si/solid-authentication
+
+## Present
+
+* Henry Story
+* Aaron Coburn
+* Dmitri Zagidulin
+* Matthieu Bosquet
+* Sarven Capadisli
+* Justin Bingham
+
+## Agenda
+
+* [Adding meeting notes from 2021-03-01](https://github.com/solid/authentication-panel/pull/144) -- (15 seconds)
+* Possible adjustments to meeting time
+* Any discussion/follow-ups from last week’s W3C Credentials WG meeting
+*  [Http Sig History missing](https://github.com/solid/authentication-panel/issues/143) (3 minutes) - what to do?
+*  [Scope discussion](https://github.com/solid/authentication-panel/issues/86)
+* Our use of the webid claim in Solid-OIDC and how we can best support DIDs
+* [Clarifying specific areas of overlap with AuthZ #145](https://github.com/solid/authentication-panel/issues/145)
+
+
+## Minutes
+
+### [Adding meeting notes from 2021-03-01](https://github.com/solid/authentication-panel/pull/144) -- (2 minutes)
+
+Todo - adjust date
+
+### Possible adjustments to meeting time
+
+EU and US times are not aligned for the next two weeks. Let's keep authentication pannel to 10am ET since the only known conflict is 11am ET.
```

Comment:
```suggestion
EU and US times are not aligned for the next two weeks. Let's keep authentication panel to 10am ET since the only known conflict is 11am ET.
```